src/app/archive/lib/processor.ml

```diff
@@ -678,15 +678,15 @@ module Zkapp_permissions = struct
     in
     ( match versions with
     | Ok [] ->
-        [%log error]
-          ~metadata:[ ("perms", Permissions.to_yojson perms) ]
-          "No transaction version exists for the permission"
+        failwith
```

Comment:
why should this fail during archiving? Are we explicitly adding the protocol version somewhere?

Reply:
We are adding protocol_version when archive the block.